akka-stream/src/main/scala/akka/stream/javadsl/OperationAttributes.scala

```diff
@@ -50,7 +50,9 @@ object OperationAttributes {
     private[akka] def asScala = scaladsl.OperationAttributes.dispatcher(dispatcher)
   }
 
-  private[akka] val none: OperationAttributes = new OperationAttributes {
+  def none() = _none
+
+  val _none: OperationAttributes = new OperationAttributes {
```

Comment:
Had to check up on this (didn't remember).
It's not really *needed*, but thought the reusing the none instance may be useful - shall I revert?